src/components/FormDay.js

```diff
@@ -0,0 +1,154 @@
+import styled from 'styled-components';
+import { useForm } from 'react-hook-form';
+import getDates from './hooks/getDates.js';
+import getDisplayDate from './hooks/getDisplayDate.js';
+import { useImmer } from 'use-immer';
+
+export default function FormDay({
+  currentDestination,
+  formName,
+  buttonName,
+  setNewDay,
```

Comment:
hey justus!really nice work so far 👍 in my last code review lene told me not to pass setters...it might be better if you use a callback function instead! for e.g. `function handleNewDay(value) { setNewDay(value)}` then you can just call your function instead to pass the setter

Reply:
It's actually not a setter function, but I see the naming of the function is not the best right there 😂